README.md

```diff
@@ -35,14 +35,19 @@ cd Actively/views/
 npm start
 ```
 
-### Running Linter
-From root folder
-...
-To run on all files
-...
-./node_modules/.bin/eslint .
-...
-To run on specific file
-...
-./node_modules/.bin/eslint File.js
-...
+
+### Setup the Linter
+```
+npm install eslint --save-dev
+./node_modules/.bin/eslint --init
+(Select Popular Style -> AirBnB -> JSON)
+```
+Running Linter
+```
+npm run lint
+```
+To run on specify file, go to package.json and change it to:
+```
+"lint": "./node_modules/.bin/eslint name_of_the_file.js; exit 0"
+```
+
```

Comment:
The linter is run against all files. Delete the specific file documentation block.